package.json

```
@@ -3,7 +3,7 @@
   "publishConfig": {
     "registry": "https://www.npmjs.com"
   },
-  "version": "45.1.1",
+  "version": "6.0.0",
```

Was passiert denn wenn die Version auf npm schon existiert?

---

Ist ein typo. 
Sollte eigentlich 46 heissen...

```suggestion
  "version": "46.0.0",
```